codeception.yml-dist

```diff
@@ -0,0 +1,10 @@
+paths:
+    tests: tests
```

Comment:
this file can stay in source as  codeception.yml, no need for dist
Basicly easier is ti setup and run tests, than better.